docs/reference/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json

```diff
@@ -79,6 +79,15 @@
                                 "soft": "The reason for failed delivery is temporary.",
                                 "hard": "The reason for failed delivery is permanent."
                             }
+                        },
+                        "https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/category": {
```

Comment:
@sauagarw : I assume this field is mandatory in case of 'feedbackStatus' is 'bounce'. If so then @prabhum2 / @mjasrotia can we define conditional required fields in XDM?

Also if for some reason producer doesn't specifies the value for this field for feedbackStatus = 'bounce', then what should consumer of this event assumes for category field?

Reply:
@sawhneyh AEP does not support defining conditions in the required filed . This needs to be handled at the data ingestion level